ovh/data_cloud_project_database_ip_restrictions.go

```diff
@@ -0,0 +1,70 @@
+package ovh
+
+import (
+	"fmt"
+	"log"
+	"net/url"
+	"sort"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/ovh/terraform-provider-ovh/ovh/helpers"
+	"github.com/ovh/terraform-provider-ovh/ovh/helpers/hashcode"
+)
+
+func dataSourceCloudProjectDatabaseIpRestrictions() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceCloudProjectDatabaseIpRestrictionsRead,
+		Schema: map[string]*schema.Schema{
+			"service_name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				DefaultFunc: schema.EnvDefaultFunc("OVH_CLOUD_PROJECT_SERVICE", nil),
+			},
+			"engine": {
+				Type:        schema.TypeString,
+				Description: "Name of the engine of the service",
+				Required:    true,
+			},
+			"cluster_id": {
+				Type:        schema.TypeString,
+				Description: "Cluster ID",
+				Required:    true,
+			},
+
+			//Computed
+			"result": {
```

Comment:
`result` is not explicit.
What about `ips` because you return a list of IP?

Example not explicit:
```
data "ovh_cloud_project_database_ip_restrictions" "iprestrictions" {
    cluster_id   = "367e1649-80d0-4338-9d82-ebc5d4caa01e"
    engine       = "postgresql"
    id           = "529713768"
    result       = [
        "178.97.6.0/24",
    ]
    service_name = "a212a1e43b614c4ba27a247b890fcf59"
}
```

Reply:
I agree with that.

For `ovh_cloud_project_database`datasource, do you prefer `clusterIds`instead of `result` ?

Reply:
Yes for `ovh_cloud_project_databases` I prefer `clusterIds` instead of result. 

Thanks :)